0chain_dev_deployment.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+sharderCount=1
+minerCount=3
+blobberCount=1
+
+while getopts s:m:b: flag
+do
+    case "${flag}" in
+        s) sharderCount=${OPTARG};;
+        m) minerCount=${OPTARG};;
+        b) blobberCount=${OPTARG};;
+    esac
+done
+
+git clone https://github.com/0chain/0chain.git
+git clone https://github.com/0chain/blobber.git
+git clone https://github.com/0chain/0dns.git
```

Comment:
Can we have a check here if these directories are present already?

Reply:
if it is already present these commands anyway do no harm
![image](https://user-images.githubusercontent.com/11261548/150424562-ae8d27c3-6d7b-4427-9a3e-ccb70686a0a6.png)